pytest.ini

```diff
@@ -10,3 +10,5 @@ norecursedirs = .* docs completion refactor absolute_import namespace_package
 # fine as long as we are using `clean_jedi_cache` as a session scoped
 # fixture.
 usefixtures = clean_jedi_cache
+
+testpaths = jedi test
```

Comment:
What do you think about moving this to setup.cfg (as `[tool:pytest]`)?
(There are ideas to deprecate setup.cfg for pytest though.)
Let's maybe just keep it here for now.

Reply:
I would probably just keep it and see how that all develops. I feel like this is such a core pytest feature and nobody really seems to be using `setup.cfg` so I would just keep it as is.